client/src/Pages/Dashboard/DashboardView.js

```diff
@@ -37,33 +40,39 @@ export default function DashboardView(props) {
     return (
       <>
         <DashboardHeader />
-        <CarList>
-
-        </CarList>
+        <CarList />
```

Comment:
needs to be in the wrapper component. (You might need to move the dashboard header up too otherwise it'll look weird)